docs/reference/rise-primitives.md

```diff
@@ -0,0 +1,37 @@
+---
+id: rise-primitives
+title: RISE Primitives
+sidebar_label: RISE Primitives
+slug: /reference/rise-primitives
+---
+
+## Type syntax cheatsheet
+
+Type | Syntax
+-----|-------
+Function from `A` to `B` | `(A -> B)`
+Dependent function from `a : A` to `B a` | `(x : A -> B)`
+Dependent function application | `f(n)`
+Vector of `A` of size `n` | `<n>A`
+Array of `A` of size `n` | `n.A`
+Dependent array of `F : Nat -> A` of size `n` | `n..F`
+Index of size `n : Nat` | `idx[n]`
+Pair of `A` and `B` | `(A, B)`
+Dependent pair of `x : A` and `B a` | `(x : A ** B)`
+
+## Primitives
+
+```scala mdoc:passthrough
+import rise.core.{primitives, makeClosed}
+println("Primitive | Type")
+println("----------|-----")
+primitives.members.foreach { p =>
+  var ts = p.typeScheme
+  // Try and close the implicit identifiers in the types
+  try { ts = makeClosed(ts)._1 }
+    catch { case _ : Throwable => () }
```

Comment:
I am not 100% sure if I get why you are doing this?

What happens if the closing of the term silently fails?

Reply:
In previous versions `makeClosed` broke for certain primitives, in which case it reverted to the unclosed version. However this seems not to be the case on current master anymore. There is still the question of whether we want closed expressions or not though.